<?xml version="1.0" encoding="UTF-8"?>
<testsuite name="ftpserver.FTPserverTest" tests="11" skipped="0" failures="0" errors="0" timestamp="2021-11-04T18:10:09" hostname="shiny" time="0.563">
  <properties/>
  <testcase name="cwd()" classname="ftpserver.FTPserverTest" time="0.257"/>
  <testcase name="pwd()" classname="ftpserver.FTPserverTest" time="0.018"/>
  <testcase name="cdup()" classname="ftpserver.FTPserverTest" time="0.011"/>
  <testcase name="list()" classname="ftpserver.FTPserverTest" time="0.08"/>
  <testcase name="pass()" classname="ftpserver.FTPserverTest" time="0.012"/>
  <testcase name="pasv()" classname="ftpserver.FTPserverTest" time="0.085"/>
  <testcase name="port()" classname="ftpserver.FTPserverTest" time="0.006"/>
  <testcase name="quit()" classname="ftpserver.FTPserverTest" time="0.007"/>
  <testcase name="retr()" classname="ftpserver.FTPserverTest" time="0.032"/>
  <testcase name="stor()" classname="ftpserver.FTPserverTest" time="0.017"/>
  <testcase name="syst()" classname="ftpserver.FTPserverTest" time="0.008"/>
  <system-out><![CDATA[220 Welcome to FTPServer
220 Welcome to FTPServer
220 Welcome to FTPServer
220 Welcome to FTPServer
200 Command OK
150 File status okay; about to open data connection
220 Welcome to FTPServer
220 Welcome to FTPServer
220 Welcome to FTPServer
220 Welcome to FTPServer
220 Welcome to FTPServer
200 Command OK
150 File status okay; about to open data connection
220 Welcome to FTPServer
200 Command OK
150 File status okay; about to open data connection
testing testing
from main dir

220 Welcome to FTPServer
]]></system-out>
  <system-err><![CDATA[]]></system-err>
</testsuite>
